### PR TITLE
Update for changes in SoE

### PR DESCRIPTION
**Pull request by cjwest**
# NOT READY FOR REVIEW

# Summary
- We removed the credits field for images on SoE. This updates the Beans so that they no longer test for the credits fields. See ticket: https://stanfordits.atlassian.net/browse/SOE-3065
This will probably fail on image issues. I have a separate ticket to fix that:
https://stanfordits.atlassian.net/browse/SOE-3066

# Needed By (Date)
- Sprint end

# Criticality
- Fixes broken stuff

# Steps to Test
- Switch to this branch: 5.x-SOE-3065-credits
- Navigate to linky_clicky/sites/engineering
- Run behat test:
 `../../bin/behat -p sites-jse-soe-stage -s dev features/engineering_stanford_bean_types`
- Verify it _doesn't_ fail with any credit checks. 
It may fail on image issues. I have a separate ticket to fix that:
https://stanfordits.atlassian.net/browse/SOE-3066

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-3065
Collections: https://stanfordits.atlassian.net/browse/SOE-3099

## Folks to notify

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: includes/bootstrap/SWSFeatureContext.php [hunk 1]
```diff
@@ -696,4 +695,22 @@ public function iShouldSeeNoDuplicateHtmlWith($arg1)
...
   }
+
+  /**
+   * @Given I am logged out
+   */
+  public function iAmLoggedOut()
+  {
+    throw new PendingException();
+  }
+
```
Comment (cjwest) [inline includes/bootstrap/SWSFeatureContext.php:706]: It's not picking up this function. Can you see what I'm missing?

  ```
 And I press the "Save" button                                                       # SWSMinkContext::pressButton()
    And I should be on "magazine/collection-behat"                                      # SWSMinkContext::assertPageAddress()
    Given the cache has been cleared                                                    # SWSDrupalContext::assertCacheClear()
    Given I am logged out
    Given I am on "magazine/article/article-2-behat"                                    # SWSMinkContext::visit()
    And I should not see "Collection for Behat"                                         # SWSMinkContext::assertPageNotContainsText()
    Given I am logged in as a user with the "administrator" role                        # SWSDrupalContext::assertAuthenticatedByRole()
    Given I am on "magazine/article/article-1-behat"                                    # SWSMinkContext::visit()
    And I click "Edit"                                                                  # SWSMinkContext::assertClick()
    And I press "Delete"                                                                # SWSMinkContext::pressButton()
    And I press "Delete"                                                                # SWSMinkContext::pressButton()
    Given I am on "magazine/article/article-2-behat"                                    # SWSMinkContext::visit()
    And I click "Edit"                                                                  # SWSMinkContext::assertClick()
    And I press "Delete"                                                                # SWSMinkContext::pressButton()
    And I press "Delete"                                                                # SWSMinkContext::pressButton()
    Given I am on "magazine/collection-behat"                                           # SWSMinkContext::visit()
    And I click "Edit"                                                                  # SWSMinkContext::assertClick()
    And I press "Delete"                                                                # SWSMinkContext::pressButton()
    And I press "Delete"                                                                # SWSMinkContext::pressButton()

1 scenario (1 undefined)
46 steps (30 passed, 1 undefined, 15 skipped)
1m46.46s (20.68Mb)

--- FeatureContext has missing steps. Define them with these snippets:

    /**
     * @Given I am logged out
     */
    public function iAmLoggedOut()
    {
        throw new PendingException();
    }

```

**Comment by minorwm** [comment]
Merging with fails: 

--- Failed scenarios:

    features/jse_editor.feature:33+
    features/jse_site_owner.feature:23
    features/jse_site_owner.feature:42
    features/jse_site_owner.feature:56

261 scenarios (256 passed, 4 failed, 1 undefined)
1619 steps (1562 passed, 4 failed, 1 undefined, 52 skipped)
55m35.38s (21.71Mb)

--- FeatureContext has missing steps. Define them with these snippets:

    /**
     * @Given I am logged out
     */
    public function iAmLoggedOut()
    {
        throw new PendingException();
    }

File: includes/bootstrap/SWSFeatureContext.php [hunk 2]
```diff
@@ -696,4 +695,22 @@ public function iShouldSeeNoDuplicateHtmlWith($arg1)
...
+  /**
```
Comment (sherakama) [inline includes/bootstrap/SWSFeatureContext.php:707]: Hey, this comment is not complete and is causing syntax issues.

File: includes/bootstrap/SWSFeatureContext.php [hunk 3]
```diff
@@ -696,4 +695,22 @@ public function iShouldSeeNoDuplicateHtmlWith($arg1)
       throw new ExpectationException($message, $session);
     }
   }
+
+  /**
+   * @Given I am logged out
+   */
+  public function iAmLoggedOut()
+  {
+    throw new PendingException();
```
Comment (sherakama) [inline includes/bootstrap/SWSFeatureContext.php:704]: Why are why changing this function to throw an error rather than leaving it to log a user out?

Comment (cjwest) [reply includes/bootstrap/SWSFeatureContext.php:704]: That wasn't my intention. I'll fix it.

**Pull request merged**